src/organisms/Status/collections/BadRequest.stories.tsx

```diff
@@ -30,4 +32,69 @@ type Story = StoryObj<typeof BadRequest>;
 
 export const Default: Story = {
   args: {},
+  play: async ({ canvas }) => {
+    // Test that default title is rendered
+    await expect(canvas.getByText('Bad request')).toBeVisible();
+
+    // Test that default description is rendered
+    await expect(
+      canvas.getByText(/We encountered some confusion regarding your request/)
+    ).toBeVisible();
+
+    // Test that back button is present by default
+    const backButton = canvas.getByRole('button');
+    await expect(backButton).toBeVisible();
+  },
+};
+
+export const CustomDescription: Story = {
+  args: {
+    description:
+      'Invalid request format. Please check your input and try again.',
+  },
+  play: async ({ canvas }) => {
```

Comment:
Nitpicky: You can do this instead so you don't have to write the same string twice

```tsx
play: async ({ canvas, args }) => {
   args.description => the same string
```